common/patches/rof2.cpp

```diff
@@ -6382,6 +6434,11 @@ namespace RoF2
 		hdr.scaled_value   = (inst->IsScaling() ? (inst->GetExp() / 100) : 0);
 		hdr.instance_id    = (inst->GetMerchantSlot() ? inst->GetMerchantSlot() : inst->GetSerialNumber());
 		hdr.parcel_item_id = packet_type == ItemPacketParcel ? inst->GetID() : 0;
+		if (item->EvolvingItem) {
+			hdr.instance_id    = inst->GetEvolveUniqueID() & 0xFFFFFFFF; //lower dword
```

Comment:
I assume this is `UINT32_MAX`? We can use `std::numeric_limits<uint32>::max()`.

Reply:
This is a mask to pull apart the 64bit.  Left it as is for now.